packages/frontend/src/hooks/chat/useMessage.ts

```diff
@@ -139,6 +140,9 @@ export default function useMessage() {
         ...message,
       })
 
+      // Notify about the sent message (listeners can filter by message type if needed)
+      notifyMessageSent(accountId, chatId, message)
```

Comment:
Introducing this front-end only event system doesn't seem like a good idea to me. I think it's better to modify core. There are a couple of problems with it:
- `notifyMessageSent` is supposed to be alongside `sendMsg` everywhere. But it's not called when sending a sticker.
- It doesn't work when you send a message from another device.
- It will require some hacks if / when we start using `sendDraft` (which BTW we shouldn't go for IMO).

But, since the work has already been done and it works, maybe it would be fine to fall back to pointing out the flaws of this system in the comments, and urging the devs not to use it.

Reply:
> but passing this just to core seems also wrong since the AppIcons are only displayed in Desktop (due to having more space)

What's wrong with that? A simple event that fires every time you sent a message sounds perfectly reasonable to me.